recipe/build.sh

```diff
@@ -14,30 +16,20 @@ export CUDA_ARCH_LIST="sm_50,sm_60,sm_70,sm_75,sm_80"
 export CUDAARCHS="50-real;60-real;70-real;75-real;80-real"
 
 if [[ "$cuda_compiler_version" == "11.2" ]]; then
-  export CUDA_ARCH_LIST="${CUDA_ARCH_LIST},sm_35"
-  export CUDAARCHS="${CUDAARCHS};35-virtual;80-virtual"
+  export CUDA_ARCH_LIST="${CUDA_ARCH_LIST},sm_35,sm_86"
+  export CUDAARCHS="${CUDAARCHS};35-real;86"
```

Comment:
Why this change? You need the `virtual` arch for newer GPUs to get better performance.

Reply:
[For CMAKE, `86` is an abbreviation of `86-real;86-virtual`.](https://cmake.org/cmake/help/latest/prop_tgt/CUDA_ARCHITECTURES.html#prop_tgt:CUDA_ARCHITECTURES)

With the extra build time, the list of target archs for 11.2 has expanded from
`35-virual;50-real;60-real;70-real;75-real;80-real;80-virtual`
to
`35-real;50-real;60-real;70-real;75-real;80-real;86-real;86-virtual`

35 was converted from virtual to real. 80 virtual was dropped, but 86 was added for both virtual and real.